src/core/thread/mle.cpp

```diff
@@ -1533,7 +1533,7 @@ Error Mle::AppendCslClockAccuracy(Message &aMessage)
 
     cslClockAccuracy.Init();
 
-    cslClockAccuracy.SetCslClockAccuracy(Get<Radio>().GetCslAccuracy());
+    cslClockAccuracy.SetCslClockAccuracy(otPlatRadioGetCslAccuracy(&GetInstance()));
```

Comment:
The `Get<Radio>().GetCslAccuracy()` calls the `otPlatRadioGetCslAccuracy`. Should we keep the code unchanged?

Reply:
I agree with Zhanglong. We should prefer calling high level functions than platform API.

Reply:
Changed.